src/test/java/org/reaktivity/nukleus/oauth/internal/control/ControllerIT.java

```diff
@@ -248,8 +253,13 @@ public void shouldResolveWithIssuerAndAudience() throws Exception
     {
         k3po.start();
 
+        JsonObject object = new JsonObject();
+        putValueIntoJsonObject(object, "issuer", "test issuer");
+        putValueIntoJsonObject(object, "audience", "testAudience");
+        String extension = object.toString();
+
         long authorization = reaktor.controller(OAuthController.class)
-                .resolve("RS256", EMPTY_STRING_ARRAY, "test issuer", "testAudience")
+                .resolve("RS256", EMPTY_STRING_ARRAY, extension)
```

Comment:
Simplify to use this approach instead.

https://github.com/reaktivity/nukleus-tls.java/blob/develop/src/test/java/org/reaktivity/nukleus/tls/internal/control/ControllerIT.java#L66